File: types/recipe.ts
interface CreateRecipe {
    title: string;
    category: string;
    steps: string[];
    authorId: number;
    description: string;
    ingredients: string[];
}

interface UpdateRecipe {
    id: number;
    title: string;
    category: string;
    steps: string[];
    authorId: number;
    description: string;
    ingredients: string[];
}

interface Recipe {
    id: number;
    title: string;
    category: string;
    steps: string[];
    authorId: number;
    description: string;
    ingredients: string[];
}

File: process-text-recipes/recipe.py
class Recipe:
    
    def __init__(self, title, description, ingredients, steps) -> None:
        self.title = title
        self.description = description
        self.ingredients = ingredients
        self.steps = steps
        
    def __str__(self) -> str:
        return f"Title: {self.title}, Description: {self.description}, Ingredients: {self.ingredients}, Steps: {self.steps}"
        